db/migrate/20210831144756_create_join_student_skills.rb

```diff
@@ -0,0 +1,8 @@
+class CreateJoinStudentSkills < ActiveRecord::Migration[6.0]
```

Comment:
same here -- student singular and skills plural
my intuition tells me it should be both plural